modules/conflict_resolver/jsx/react.resolved_conflicts.js

```diff
@@ -0,0 +1,250 @@
+import {Tabs, TabPane} from 'Tabs';
+import FilterForm from 'jsx/FilterForm';
+import formatColumn from './resolved_conflicts_columnFormatter';
+
+/**
+ *	This file contains the React classes for conflicts resolved
+ * 	module.
+ */
+
+/**
+ * @doc function
+ * @name updateState
+ * @param {object} filter to update the state.
+ * @description This function is responsible for updating
+ * the ResolvedConflictsPane state from the parent.
+ */
+var updateState = function(filter) {
+  this.setState({filter});
+};
+
+/**
+ *	This is the React class for building the instrument
+ */
+class ResolvedConflictsPane extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      Data: this.props.data
+    };
+    updateState = updateState.bind(this);
```

Comment:
Why is `updateState` being bound if the function is not used in this component?

Reply:
@HenriRabalais See previous comment similar to this.